app/views/content/life-as-a-teacher/change-careers/how-to-change-careers-to-become-a-teacher/_article.html.erb

```diff
@@ -53,3 +53,14 @@
     text: "An adviser with years of teaching experience can help you change careers to become a teacher. Chat by phone, text, or email as little or often as you need.",
   ) %>
 </div>
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render CallsToAction::MailinglistBlockComponent.new %>
```

Comment:
I'd suggest switching the order to advisers (most important for this audience), then mailing list, then routes

But also I think events should be included - important for career changers - maybe instead of mailing list